md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BusDomainAPI.java

```diff
@@ -174,8 +166,9 @@ RestWrapper update(@ModelAttribute("busdomain")
             // BusDomain busDomains = s.selectOne("call_procedures.UpdateBusDomain", busDomain);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)